tekton/cronjobs/dogfooding/nightly-tests/ppc64le/catalog-nightly-test/cronjob.yaml

```diff
@@ -19,7 +19,7 @@ spec:
             - name: NAMESPACE
               value: "bastion-p"
             - name: REGISTRY
-              value: "ppc64le-cluster.bastion-p.svc.cluster.local:443"
+              value: "na.artifactory.swg-devops.com/sys-linux-power-team-ftp3distro-docker-images-docker-local/tekton"
```

Comment:
```sh
$ crane ls na.artifactory.swg-devops.com/sys-linux-power-team-ftp3distro-docker-images-docker-local/tekton
2024/02/07 09:26:17 No matching credentials were found for "na.artifactory.swg-devops.com"
Error: reading tags for na.artifactory.swg-devops.com/sys-linux-power-team-ftp3distro-docker-images-docker-local/tekton: GET https://na.artifactory.swg-devops.com/artifactory/api/docker/null/v2/token?scope=repository%3Asys-linux-power-team-ftp3distro-docker-images-docker-local%2Ftekton%3Apull&service=na.artifactory.swg-devops.com: : Authentication is required
```

This looks like an internal IBM artifactory instance and won't be accessible to the dogfooding cluster.

Reply:
@wlynch Yes it's an internal IBM artifactory which is only accessible to the ppc64le test cluster.  It's used to build and store the images built while the tekton tests run.